publish2cloud.py

```diff
@@ -275,6 +277,15 @@ def publish_to_cloud(config, chunknum, check_versioning=None):
 
         with open(config.get(section, 'output'), 'rb') as blob:
             new = chunk_metadata(blob)
+            file_size = int(new['len']) + HEADER_SIZE
+            if file_size >= MAX_CHUNK_SIZE:
+                exceeded_max_size_msg = ('!!! {} NOT UPLOADED. '
+                    'File size is {} which exceeded max size {} !!!')
+                print(exceeded_max_size_msg.format(
```

Comment:
Is there a reason to use `print` instead of throwing an error? With print we will still upload lists which don't exceed the limit, but what if the changes are linked across lists. Let's say we add new domains to the blocklist which should have corresponding entitylist updates, but the entity list exceeds the chunksize. We'd upload the new blocklist but not the new entitylist, which might lead to breakage. Thus. it seems safer to throw an error. Are there any downsides to throwing an error?

Reply:
No downsides I could think of. Though now I think about this, we should probably make the size be dependent on the version as well right? Since Nightly (73) will have the increased chunk size while the Beta and Release will still have the older max chunk size.

Reply:
If it's not too much work that would be safest.